pkgs/by-name/yg/ygege/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  git,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ygege";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "UwUDev";
+    repo = "ygege";
+    rev = "v${version}";
+    hash = "sha256-/MBZacukK6hicLXFCWDQrP3Sec9Mgemw+5gJkRq0CmQ=";
+  };
+
+  cargoHash = "sha256-1PyJJB3CEldKHAUCvJDhXrQY2WQn8TGNVtAKvv86Y2k=";
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    git
+    rustPlatform.bindgenHook
+  ];
+
+  doCheck = false;
```

```suggestion
  doCheck = false;
  
  passthru.updateScript = nix-update-script { };
```
Nice to have, https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#automatic-package-updates